drivers/ever-hid.c

```diff
@@ -0,0 +1,638 @@
+/* HID2NUT lookup table                                            */
+/* --------------------------------------------------------------- */
+
+static hid_info_t ever_hid2nut[] = {
+
+  { "ups.batterysystem.battery.batteryid", 0, 0, "UPS.BatterySystem.Battery.BatteryID", NULL, "%.0f", 0, NULL },
+  { "ups.batterysystem.batterysystemid", 0, 0, "UPS.BatterySystem.BatterySystemID", NULL, "%.0f", 0, NULL },
+  { "ups.batterysystem.charger.chargerid", 0, 0, "UPS.BatterySystem.Charger.ChargerID", NULL, "%.0f", 0, NULL },
+  { "ups.batterysystem.input.flowid", 0, 0, "UPS.BatterySystem.Input.FlowID", NULL, "%.0f", 0, NULL },
+  { "ups.batterysystem.input.inputid", 0, 0, "UPS.BatterySystem.Input.InputID", NULL, "%.0f", 0, NULL },
+  { "ups.batterysystem.output.flowid", 0, 0, "UPS.BatterySystem.Output.FlowID", NULL, "%.0f", 0, NULL },
+  { "ups.batterysystem.output.outputid", 0, 0, "UPS.BatterySystem.Output.OutputID", NULL, "%.0f", 0, NULL },
+  /*{ "ever.identyfication.upstype", 0, 0, "UPS.EVER1.EVER12", NULL, "%s", 0, ever_format_model },*/
+  { "ever.identyfication.firmware.inverter.version", 0, 0, "UPS.EVER1.EVER13", NULL, "%s", 0, ever_format_version },
+  { "ever.identyfication.firmware.interfaces.version", 0, 0, "UPS.EVER1.EVER14", NULL, "%s", 0, ever_format_version },
+  { "ever.identyfication.hardware.version", 0, 0, "UPS.EVER1.EVER15", NULL, "%s", 0, ever_format_hardware },
+  { "ever.identyfication.protocol.inverter.version", 0, 0, "UPS.EVER1.EVER16", NULL, "%s", 0, ever_format_version },
+  { "ever.identyfication.protocol.interfaces.version", 0, 0, "UPS.EVER1.EVER17", NULL, "%s", 0, ever_format_version },
+  { "ever.hitsinktemperature", 0, 0, "UPS.EVER1.EVER42", NULL, "%s", 0, kelvin_celsius_conversion },
+  { "ever.batterytemperature", 0, 0, "UPS.EVER1.EVER43", NULL, "%s", 0, kelvin_celsius_conversion },
+  { "ever.outputpowerfactor", 0, 0, "UPS.EVER1.EVER44", NULL, "%.0f", 0, NULL },
+  { "ever.ups.control.upson", ST_FLAG_RW | ST_FLAG_NUMBER, 0, "UPS.EVER1.EVER45.EVER46", NULL, "%.0f", 0, NULL },
+  { "ever.ups.control.clearfault", ST_FLAG_RW | ST_FLAG_NUMBER, 0, "UPS.EVER1.EVER45.EVER47", NULL, "%.0f", 0, NULL },
+  { "ever.ups.control.clearbatteryfault", ST_FLAG_RW | ST_FLAG_NUMBER, 0, "UPS.EVER1.EVER45.EVER48", NULL, "%.0f", 0, NULL },
+  { "ever.ups.control.epoblocked", ST_FLAG_RW | ST_FLAG_NUMBER, 0, "UPS.EVER1.EVER49.EVER50", NULL, "%.0f", 0, NULL },
+  { "ever.ups.control.greenmode", ST_FLAG_RW | ST_FLAG_NUMBER, 0, "UPS.EVER1.EVER49.EVER51", NULL, "%.0f", 0, NULL },
+  { "ever.ups.control.buttonsound", ST_FLAG_RW | ST_FLAG_NUMBER, 0, "UPS.EVER1.EVER49.EVER52", NULL, "%.0f", 0, NULL },
+  { "ever.ups.control.audiblealarm", ST_FLAG_RW | ST_FLAG_NUMBER, 0, "UPS.EVER1.EVER49.EVER53", NULL, "%.0f", 0, NULL },
+  { "ever.ups.config.outputvoltage", ST_FLAG_RW | ST_FLAG_STRING, 3, "UPS.EVER1.EVER54", NULL, "%.0f", 0, NULL },
+  /*{ "ever.ups.config.minoutputvoltage", ST_FLAG_RW | ST_FLAG_STRING, 3, "UPS.EVER1.EVER55", NULL, "%.0f", 0, NULL },
```

Comment:
For variables that are commented out, it would be useful to know why. Is this not implemented in this UPS?

Reply:
Updated later: Line removed from current code